CarInspection/src/WebCarInspection/wwwroot/lib/bootstrap/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
I described this issue in our meetings.
Lib folder should absent in the repo and libman config should be created.
Ask Vladimir Zaitsev for consultancy.